.eslintrc.json

```diff
@@ -55,7 +55,7 @@
     {
       "files": ["src/**/*"],
       "plugins": ["react-hooks"],
-      "extends": ["plugin:react/recommended"],
+      "extends": ["plugin:react/recommended", "plugin:jest-dom/recommended"],
```

Comment:
Can we scope this plugin to tests only?

Reply:
I was thinking about this, but realized that we have other plugins like React that don't make sense for all the files anyway. Want me to make a PR splitting it up further?

Reply:
Yeah, adding more fine-grained `overrides` sounds great!